packages/commonwealth/client/scripts/views/pages/CommunityManagement/Contests/ContestsList/ContestCard/ContestCard.tsx

```diff
@@ -119,7 +147,9 @@ const ContestCard = ({
               <CWText className="label">
                 {moment.localeData().ordinal(index + 1)} Prize
               </CWText>
-              <CWText fontWeight="bold">{s.tickerPrize} ETH</CWText>
+              <CWText fontWeight="bold">
+                {capDecimals(s.tickerPrize!.toFixed(decimals))} {ticker}
```

Comment:
is the `!` intentional here?

Reply:
Assuming we want to eventually fix null check errors, I think `// @ts-expect-error <StrictNullChecks/>` is better compared to modifying logical areas with `!`, this would also be easier to remove later on, while finding each `!` and removing it would be harder.

I also don't have the full context to null checks here, so not sure if this makes sense. Please resolve if not.

Reply:
actually, I would prefer to add comment rather than `!` exactly for reason of easier replacement in the future. Let's try to confirm with @burtonator

Reply:
Well, one thing to think about is that we *already* have "!" in place.  That's what happens by default when strictNullChecks is false.  

In most situations we'd get a null pointer exception there.

I'm fine with keeping // @ts-expect-error <StrictNullChecks/> if we want to use that as a marker for code that needs to get refactored.  That's probably a good idea.

There are fallback conditions we haven't talked about yet.  For example, if a date isn't present, how do we handle that situation?  We still have to do the work to figure that out.

Also, I think that many of our null issues are false.  They're not really nullable because we would have seen errors by now. 

Honestly though, since these would normally generate an *error* we're pretty flexible with how we handle them. Anything is better than an error.

Also, I guess we don't have to discuss with the platform team but if we agree on this we should probably notify the team as PENG isn't the only team updating the frontend.

Note that the <StrictNullChecks/> was used to find incorrect TSX comments when because the that would trigger a TSX compilation error if it was used in the wrong context.  We can also just use "StrictNullChecks". It's really just a string to note thy the @ts-expect-error was there.

Reply:
thanks @burtonator for your answer. So I would opt in for using `// @ts-expect-error <StrictNullChecks/>` instead of `!`, of course as a last resort. We should try to do everything to make types work correctly